### DISCUSSION TITLE
[疑问] 青衣前辈拍照水平怎么样呢

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1247] [其他]
![Screenshot_20240811_070425](https://github.com/user-attachments/assets/9ce29445-e3d3-4b47-bdc7-9f683198c0bf)
早，新艾利都，今天来海边执勤，前辈说要拍我当治安局宣传海报，还必须要我换上泳装。怎么说呢……总感觉前辈拍得有点奇怪，这样的照片真的能吸引公众吗…总有些不信。

海边还是很美的（附上图片
![wx_camera_1723263329230](https://github.com/user-attachments/assets/098993d7-c016-4ba5-bf95-581fbeab1a2d)
）

**Comment by sishui-buyin** [comment]
朱鸢小姐真好看owo

**Comment by ZhuYuan-148** [comment]
![Screenshot_20240811_114437](https://github.com/user-attachments/assets/bad8dcd6-ae8e-4368-8466-bdbadebd73fc)
最终选取这张喽，感谢各位市民的支持

**Reply by beibei-b** [reply]
好可爱的前辈，想rua🤤

**Reply by ZhuYuan-148** [reply]
这位市民，请收起袭警的念头。

**Comment by Qiling-XwX** [comment]
立啦！
